src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
@dmethvin 
> I had the same confusion about why msFullscreenElement was being (re)checked but then I read the docs and I assume this variable is there but null if we're not currently in fullscreen.

That's the usual idiom for DOM stuff. :) A property always exists and is just set to `null` (not `undefined`) if not defined. The same applies for e.g. `history.state`, you can just check if it's `!== undefined` to be sure it's supported.